server/job/job.js

```diff
@@ -261,11 +261,21 @@ class Job {
     return this.getContextProp(Job.keysContext.survey)
   }
 
+  //TODO replace it with surveyId getter
   getSurveyId () {
+    return this.surveyId
+  }
+
+  get surveyId () {
     return this.getContextProp(Job.keysContext.surveyId)
   }
 
+  //TODO replace it with user getter
   getUser () {
+    return this.user
+  }
+
+  get user () {
     return this.getContextProp(Job.keysContext.user)
   }
 
```

Comment:
I'm not sure i like those TODOs. we should use the old way or take care of this after this pull request. what do you think?

Reply:
we can do a search and replace after this pull request, it makes the code a bit more readable

Reply:
yes i agree, please add a Trello issue